sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -237,6 +252,7 @@ public partial class ServiceBusProcessor : System.IAsyncDisposable
         public virtual bool AutoCompleteMessages { get { throw null; } }
         public virtual string EntityPath { get { throw null; } }
         public virtual string FullyQualifiedNamespace { get { throw null; } }
+        public string Identifier { get { throw null; } }
```

Comment:
I think we should probably make this virtual, if for no other reason than to be consistent with the other properties. Not all of them needed to be virtual in the first place, but I think it will be more annoying for devs if some things can be mocked and others can't.

Reply:
I disagree with the need; this property can be directly set by passing options.   Generally we make things virtual when callers can't directly control them.    That said, I don't feel strongly about it, just seems to cut against the normal pattern.

Reply:
Yeah, as I mentioned the other properties didn't need to be virtual in the first place, but I think it will be a bad user experience if only some of the properties, but not all of them, can be mocked.